File: 4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {

        double ans;
        for(int i=0; i<nums2.size(); i++)
            nums1.push_back(nums2[i]);
            sort(nums1.begin(), nums1.end());
            int s = 0, e = nums1.size()-1;
            int mid = s + (e-s)/2;
            if(e%2==0)
            {
            ans = nums1[mid];
            }
            else
            {
                ans = nums1[mid] + nums1[mid+1];
                ans=ans/2;
            }
        return ans;
        
    }
};

File: 4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {

        int m = nums1.length;
        int n = nums2.length;
        int l = m + n;
        int arr[] = new int[l];
        int i = 0;
        int j = 0;
        int k = 0;

        while (i < m && j < n) {
            if (nums1[i] <= nums2[j]) arr[k++] = nums1[i++];
            else if (nums1[i] > nums2[j])  arr[k++] = nums2[j++];
        }

        while (i < m) arr[k++] = nums1[i++];
        while (j < n) arr[k++] = nums2[j++];

        if (arr.length % 2 != 0) return (double) arr[arr.length / 2];
        else {
            int mid = arr.length / 2;
            return ((double)(arr[mid] + arr[mid - 1]) / 2);
        }

    }
}
